internal/integrationtest/apigateway/smoke_test.go

```diff
@@ -1,6 +1,6 @@
 // +build integration
 
-package apigateway
+package directconnect
```

Comment:
A little accident here I suppose.

Probably just need to update the file name to directconnect/smoke_test.go